examples/core/core_custom_frame_control.c

```diff
@@ -27,12 +27,20 @@
 ********************************************************************************************/
 
 #include "raylib.h"
+#include "config.h"
+#include "assert.h"
```

Comment:
Since this is a system header it should use <> not ""

Reply:
true :+1: